index.rst

```diff
@@ -437,284 +329,111 @@ this case, we may have something like:
-version that matches the one used by the CSC at that time, check the
-``_label.yaml`` file so they can finally find out what configuration was used.
-
-This information should probably be present in both ``settingVersions`` and
-``settingsApplied``. For instance ``settingVersions`` should contain an
-additional entry, e.g. ``mapping``, with the names of the files the labels
-maps to (in case of git-based configuration) or any other appropriate mapping
-information (e.g. for database configuration). Then, ``settingsApplied`` should
-also have an entry for ``label`` and ``mapping`` that will contain information
-about the selected label.
-
-.. _section-appendix-available-vs-selected:
-
-Available vs. selected
-----------------------
+#.  Rename ``settingsVersions`` to ``configurationsAvailable``
 
-The ``settingsApplied`` and ``settingsVersion`` events represent what is
-available and what was selected, respectively. These topics should have
-*nearly* identical attributes; available labels vs selected label, available
-mapping vs selected map, available branch description vs. selected branch
-description and so on.
+    #.  Rename ``recommendedSettingsLabels`` to ``labels``.
+    #.  Rename ``recommendedSettingsVersion`` to ``versions``.
+    #.  Rename ``settingsUrl`` to ``url``.
+    #.  Add ``mapping``.
+        For a configuration repository ``mapping`` will reflect the content of the ``_labels.yaml`` file.
+        For a configuration database ``mapping`` will explicitly show the mapping of ``label`` to ``label:version``.
 
-.. _section-appendix-renaming:
+#. Rename ``settingsApplied`` to ``configurationApplied``
 
-Topic renaming
---------------
+    #.  Add ``label``.
+    #.  Add ``version``.
+    #.  Add ``mapping``.
+    #.  Add ``url``.
+    #.  Rename ``otherSettingsEvent`` -> ``otherInfo``.
 
-The interface would be much clear if we rename some topics and attributes to
-reflect more closely their true meaning.
+  The event will publish the selected values once the CSC is configured.
 
-The following is a renaming suggestion for discussion:
+#. In the ``start`` command, rename attribute ``settingsToApply`` to ``configuration``.
+
+.. _section-continuous-monitoring:
+
+Continuous monitoring configuration repository
+----------------------------------------------
+
+Right now CSCs are required to publish ``configurationsAvailable``  (former ``settingsVersions``, see :ref:`renaming proposal <section-renaming>`) when they transition to ``STANDBY`` state.
+Nevertheless, while in ``STANDBY`` state it is possible for someone to update the available configuration, which would make the information out of sync.
+We propose that, while in ``STANDBY`` state, CSCs continuously monitor the configuration repository and update the information if needed.
+If an invalid configuration is made available while the CSC is in ``STANDBY`` the CSC should transition to ``FAULT`` state.
```

Comment:
with an appropriate error message saying which file caused the issue and how to resolve it.

Reply:
might also add that this monitor will not apply when in any other state.

Reply:
I did say "while in `STANDBY`" but I'll reinforce it...